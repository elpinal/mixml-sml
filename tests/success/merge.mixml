link U =
  {module U =
      link X = {module t = [type int]} with
      link Y = {module u = [type :Type]} with
      {module f = [val :ext Y.u]}
  }
with
  {module U =
      link X = {module t = [type :Type]} with
      link Y = {module u = [type bool]} with
      {module f = [val true]}
  }
