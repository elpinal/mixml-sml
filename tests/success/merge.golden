{U : {f : [bool]+, t : [= int], u : [= bool]}}
